backend/main/others/dynamodbInitializer/src/index.ts

```diff
@@ -90,6 +94,22 @@ const input: CreateTableCommandInput = {
 				WriteCapacityUnits: 1,
 			},
 		},
+		{
+			IndexName: "EMailAndUserIdRelationship",
+			KeySchema: [
+				{
+					AttributeName: "EMailAddress",
+					KeyType: "HASH",
+				},
```

Comment:
Don't we need UserId as SortKey in this GSI?
```
{
	AttributeName: "UserId",
	KeyType: "RANGE",
},
```

Reply:
I do not think so.
The email address must be unique on its own. Therefore, it doesn't need additional data.

Reply:
I'm so sorry. I misunderstood. Now I got why the UserId isn't needed. Thank you!